test/upgrade/prober/wathola/event/services.go

```diff
@@ -105,9 +113,21 @@ func (f *finishedStore) State() State {
 	return f.errors.state
 }
 
-func (f *finishedStore) Thrown() []string {
+func (f *finishedStore) DuplicatedThrown() []string {
+	return f.thrownHelper(f.errors.duplicatedThrown)
+}
+
+func (f *finishedStore) MissingThrown() []string {
+	return f.thrownHelper(f.errors.missingThrown)
+}
+
+func (f *finishedStore) UnexpectedThrown() []string {
+	return f.thrownHelper(f.errors.unexpectedThrown)
+}
+
+func (f *finishedStore) thrownHelper(errThrown []thrown) []string {
```

Comment:
Use meaningful name, rather than `thrownHelper`. 

For example: `asStrings(errs []thrown) []string` seams more meaningful to me.

Reply:
Fixed as suggested.